Cargo.toml

```diff
@@ -126,3 +126,8 @@ path = "examples/form_data/form_data.rs"
 
 name = "https"
 path = "examples/https.rs"
+
+[[example]]
+
+name = "server_with_data"
+path = "examples/server_with_data.rs"
```

Comment:
Could this be renamed to something that makes it clear that there's no macro use? Perhaps your branch name `no_macro_custom_data`?